.github/workflows/sweep.yml

```diff
@@ -0,0 +1,23 @@
+name: Test Sweep
```

Comment:
This is a useful github action that can be ran only on demand through the Github UI in case the test tenant gets corrupted.

Reply:
Something like that yes, it basically sweeps through the tenant resources and cleans up anything the tests might have created and not cleaned up properly after the test ran. 

And yes it's a completely idempotent process and it will only affect resources created from the tests. It won't touch anything else manually created on the tenant so it's pretty safe to run.